src/main/java/decodes/xml/PlatformConfigParser.java

```diff
@@ -24,8 +27,8 @@ public class PlatformConfigParser implements XmlObjectParser, XmlObjectWriter, T
 	/**
 	 * Reference to the DecodesScriptBuilder so we can finalized after we've read it in.
 	 */
-	private DecodesScriptBuilder scriptBuilder; 
-	private DecodesScriptParser scriptParser;
+	private List<DecodesScriptParser> decodesScriptParserList = new ArrayList<>();
+	private List<DecodesScriptBuilder> decodesScriptBuilderList = new ArrayList<>();
```

Comment:
This looks like a reasonable solution to me. Reading through it I suspect we could limit this to just an array of `DecodeScriptBuilder`s. I think my original work in this design it seemed impractical to do so.

But that can be addressed later.